website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,554 @@
+* What permissions you need to set up if you are adding Karpenter to an existing cluster
+
+# Review the cloudformation.yaml file
+
+To download a particular version of `cloudformation.yaml`, set the version and use `curl` to pull the file to your local system:
+
+```bash
+export KARPENTER_VERSION=v0.29.0
+curl https://raw.githubusercontent.com/aws/karpenter/"${KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml > cloudformation.yaml
+```
+
+The `cloudformation.yaml` file starts with the following information:
+
+```
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/aws/karpenter
+Parameters:
+  ClusterName:
+    Type: String
+    Description: "EKS cluster name"
+Resources:
+```
+
+The rest of the `cloudformation.yaml` file describes the resources that CloudFormation deploys.
+The sections of that file can be grouped together under the following general headings:
+
+* **Node Authorization**: Creates a NodeInstanceProfile, attaches a NodeRole to it, and connects it to an IAM Identity Mapping that Karpenter uses. This defines the permissions each node managed by Karpenter has to access EC2 and other AWS resources.
+* **Karpenter Controller Authorization**:  Creates a service account (named `karpenter`) that is combined with the KarpenterControllerPolicy. The KarpenterControllerPolicy allows the Karpenter controller to control assets within EC2 specifically and within AWS in general.
+* **Interruption Handling**: The interruption handling sections of this file allow the Karpenter controller to see and respond to interruptions that occur with the nodes that Karpenter is managing. Interruptions can reflect things like a spot instance going away or physical hardware going down. Allowing the Karpenter controller to see these interruptions allows Karpenter to respond by bringing nodes up and down and moving workloads.
+
+A lot of the object naming that is done by `cloudformation.yaml` is based on the following:
+
+* Cluster name: With a user name of `bob` the Getting Started Guide would name your cluster `bob-karpenter-demo`
+That name would then be appended to any name below where `${ClusterName}` is included.
+
+* Partition: Any time an ARN is used, it includes the partition name to identify where the object is found. In most cases, that partition name is `aws`. However, it could also be `aws-cn` (for China Regions) or `aws-us-gov` (for AWS GovCloud US Regions).
+
+# Node Authorization 
+
+The following sections of the `cloudformation.yaml` file set up permissions related to what Kubernetes nodes created by Karpenter can do with EC2 and other AWS features.
+In particular, this involves setting up an instance profile and attaching a node role to that profile with the following objects:
+
+* KarpenterNodeInstanceProfile
+* KarpenterNodeRole
```

Comment:
I'd list the role first, then the instance profile that depends on it.

Reply:
Done.